src/main/java/com/example/LoyaltyCard.java

```diff
@@ -0,0 +1,29 @@
+package com.example;
+
+import static com.google.common.base.Preconditions;
+
+
+/**
+ * Class implements Value Object.
+ */
+public class LoyaltyCard {
```

Comment:
Most likely the class is `final`. Assume all the classes you create are `final`, unless you specifically need to plan for inheritance.

Reply:
@elena-yevs, any thoughts on this my comment?